### DISCUSSION TITLE
Inconsistency on the display of `discussion` in the list of personal activities

### DISCUSSION THREAD

**Discussion by xuyxu** [community/community#3241] [Discussions] [answered]
Hi, thanks for bringing this new feature. I think it is quite helpful, and enables users and developers to have better interactions ;-)

Here is something that I think could be improved: The activity on my discussion is displayed on the list of my activities on other groups. The figure below shows that my discussion in `pytest-dev` is displayed on my activities in `sktime`, which should only be displayed when clicking the badge `pytest-dev`, right?

![tmp](https://user-images.githubusercontent.com/22359569/105938723-c2863880-6092-11eb-8bb4-e903e405b3f7.png)

Once again, thanks for bringing this feature to GitHub, which helps me a lot!

**Comment by PranamBhat** [comment, marked as answer]
It might be due to session timeout. Try refreshing the page and clicking the specific tab one more time.
